libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/AdminInformationMessage.tsx

```diff
@@ -21,7 +21,7 @@ export const AdminInformationMessage: React.FC<IAdminInformationMessageProps> =
             className="gd-granular-permissions-admin-information s-granular-permissions-admin-information"
             onClose={onClose}
         >
-            <span>
+            <span aria-label="Share dialog admin information message">
```

Comment:
Should aria-labels be translated?

Reply:
In general terms of accessibility, yes, but I would skip it for now it will be handled in the future and is a larger topic. Also there already are a lot of aria-labels from former test rewrites as RTL enforces such labels instead of selectors.